src/TrendsOverview.svelte

```diff
@@ -206,6 +220,37 @@
         return "";
     }
   }
+
+  function goToParentRegion(): void {
+    let parentRegion: Region;
+
+    if (isSubRegionOne(selectedRegion)) {
```

Comment:
Nit: It may simplify logic to add a enum to `Region` to say which geo type it is, and we can populate it when we parse out the initial CSV.

Reply:
E.g. `region.type = RegionType.SubRegion1`.  Then we don't have separate logic for things defined in `data.ts` and functions in `utils.ts`